lib/infra_conversion_throttler.rb

```diff
@@ -20,25 +20,41 @@ def self.pending_conversion_jobs
     pending.group_by { |job| job.migration_task.destination_ems }
   end
 
+  # @return [Hash] the list of jobs in state 'running', grouped by conversion host
```

Comment:
Could the state also be 'migrate' here?

Reply:
Nope. `migrate` state is on the ServiceTemplateTransformationPlanTask, not the InfraConversionJob.